eval/bundled/epm.elv.go

```diff
@@ -1,111 +1,279 @@
 package bundled
 
-const epmElv = `
+const epmElv = `# Verbosity configuration
+debug-mode = $false
+
+# Configuration for common domains
+-default-domain-config = [
+  &"github.com"= [
+    &method= git
+    &protocol= https
+    &levels= 2
+  ]
+  &"bitbucket.org"= [
+    &method= git
+    &protocol= https
+    &levels= 2
+  ]
+  &"gitlab.com"= [
+    &method= git
+    &protocol= https
+    &levels= 2
+  ]
+]
+
+# Internal configuration
 -data-dir = ~/.elvish
--installed = $-data-dir/epm-installed
+-lib-dir = $-data-dir/lib
 
-fn -info [text]{
-    print (edit:styled '=> ' green)
+# Runtime state - records get copied from -default-domain-config
+# or read from epm-domain.cfg files as needed
+-domain-config = [&]
```

Comment:
I think it's better to **not** cache domain configurations. Caching means that if the user edits a domain config, the change may or may not take effect, depending on whether that particular config has been cached. Just read them every time.

Reply:
Makes sense, working on it. I was also thinking: does it makes sense to create the file the first time with the default config, for known domains? If we don't, then we can just return the default config every time, which makes it easier to change the default configs in the module. It would also simplify the code: either read-and-return, or just return the default. No creating files.

Reply:
Aha, good point. I agree that we don't have to write out the default config.